.travis.yml

```diff
@@ -1,3 +1,10 @@
----
 language: ruby
-script: bundle && bundle exec rspec
+rvm:
+  - 2.4
+  - 2.5
+  - 2.6
+script: bundle exec rspec
+sudo: false
+cache: bundler
+before_install:
+  - gem install bundler
```

Comment:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
Ugh you caught me leaning on rubocop

Reply:
(Fixed)